### PR TITLE
Sherlock example meshing settings, solve, post

### PR DESCRIPTION
**Pull request by flamingto**
I updated the original Sherlock example with global mesh settings, solve and post-processing step. The original one seems to have convergence issue.

### CODE REVIEW

File: examples/04-Icepak/Sherlock_Example.py [hunk 1]
```diff
@@ -181,6 +194,30 @@
 
 ipk.assign_priority_on_intersections()
 
+###############################################################################
+# Compute power Budget
```
Comment (MaxJPRey) [inline examples/04-Icepak/Sherlock_Example.py:198]:
```suggestion
# Compute Power Budget
```

Comment (MaxJPRey) [reply examples/04-Icepak/Sherlock_Example.py:198]: or:
```suggestion
# Compute power budget
```

File: examples/04-Icepak/Sherlock_Example.py [hunk 2]
```diff
@@ -181,6 +194,30 @@
...
+###############################################################################
+# Get Solution Data and Plot
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+face_u10 = [i.id for i in ipk.modeler["COMP_U10"].faces]
+plot1 = ipk.post.create_fieldplot_surface(face_u10, "SurfTemperature")
+plot1.export_image(os.path.join(ipk.working_directory, "temperature_u10.jpg"))
+face_u9 = [i.id for i in ipk.modeler["COMP_U9"].faces]
+plot2 = ipk.post.create_fieldplot_surface(face_u9, "SurfPressure")
+
```
Comment (MaxJPRey) [inline examples/04-Icepak/Sherlock_Example.py:219]: Maybe we could export the image the same way we export the image for the temperature around `u10`.

File: examples/04-Icepak/Sherlock_Example.py [hunk 3]
```diff
@@ -181,6 +194,30 @@
...
+###############################################################################
+# Compute power Budget
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+power_budget, total = ipk.post.power_budget("W" )
+print(total)
+
+###############################################################################
+# Compute power Budget
+# ~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (maxcapodi78) [inline examples/04-Icepak/Sherlock_Example.py:206]: all title should have a number of simbols under it equal to the size of the line
```suggestion
# Compute power Budget
# ~~~~~~~~~~~~~~~~~~~~
```

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]
Please apply to all titles my suggestion

File: examples/04-Icepak/Sherlock_Example.py (hunk 3 above)
Comment (PipKat) [reply examples/04-Icepak/Sherlock_Example.py:206]: Also, the first letter of the first word in a title is capitalized, but all other letters in the title that are not proper nouns are lower case. Here, "Budget" should be "budget".

**Review by PipKat — APPROVED** [review]
Looks great to me!

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**